### ISSUE TITLE
Main container height insufficient for form content

### ISSUE THREAD

**Issue by Salvatos** [BrandonEchols04/kanka-ruler-calculation#1]
![Capture d’écran_2023-08-26_22-45-49](https://github.com/BrandonEchols04/kanka-ruler-calculation/assets/1753500/d98dae6c-5db8-4b7d-8de0-cfb19c2310dd)
![Capture d’écran 2023-08-26 à 22 59 45](https://github.com/BrandonEchols04/kanka-ruler-calculation/assets/1753500/3e3f41ea-a2f7-439b-ba64-de771a776754)

`div.parent` has a set width of 50vh and doesn't expand/scroll to have room for all content on shorter monitors. Makes it unusable on mobile since the inputs aren't even on screen :)

**Comment by BrandonEchols04** [comment]
> ![Capture d’écran_2023-08-26_22-45-49](https://user-images.githubusercontent.com/1753500/263505775-d98dae6c-5db8-4b7d-8de0-cfb19c2310dd.png) ![Capture d’écran 2023-08-26 à 22 59 45](https://user-images.githubusercontent.com/1753500/263505862-3e3f41ea-a2f7-439b-ba64-de771a776754.png)
> 
> `div.parent` has a set width of 50vh and doesn't expand/scroll to have room for all content on shorter monitors. Makes it unusable on mobile since the inputs aren't even on screen :)

So, in theory, removing `height: 50vh;` from the CSS, should fix it?

**Comment by Salvatos** [comment]
Yes, though the layout would need several other changes to properly scale to different resolutions and especially mobile.
